### PR TITLE
Rename Floonet to Testnet

### PR DESCRIPTION
**Pull request by quentinlesceller**
Equivalent of https://github.com/mimblewimble/grin/pull/3431. Tests won't pass until merged.
There is no migration setup so user will need to move/restore in the new `test` dir.

### CODE REVIEW

**Comment by quentinlesceller** [comment]
Tested and works with https://github.com/mimblewimble/grin/pull/3431

File: util/Cargo.toml [hunk 1]
```diff
@@ -35,20 +35,20 @@ sha3 = "0.8"
 # grin_store = { git = "https://github.com/mimblewimble/grin", tag = "v4.0.0-rc.1" }
 
 # For bleeding edge
-grin_core = { git = "https://github.com/mimblewimble/grin", branch = "master" }
```
Comment (antiochp) [inline util/Cargo.toml:38]: Careful we don't commit these changes and revert this file (and lock file) before merging this.

**Review by antiochp — APPROVED** [review]
I think this looks good. :+1:

File: util/Cargo.toml (hunk 1 above)
Comment (quentinlesceller) [reply util/Cargo.toml:38]: Yep that what I told you I was about to remove on Keybase

**Comment by quentinlesceller** [comment]
Thanks @antiochp I will wait until tests pass since https://github.com/mimblewimble/grin/pull/3431 is merged.

**Comment by quentinlesceller** [comment]
Unsure why tests aren't passing, master does have the `Testnet` in it.

**Comment by antiochp** [comment]
> Unsure why tests aren't passing, master does have the `Testnet` in it.

You need to `cargo update -p grin_core` to get Cargo.lock updated to latest sha on master.

**Comment by quentinlesceller** [comment]
Tests are passing. Merging. End of floonet :)

**Pull request merged**